packages/queries/src/types/Where.ts

```diff
@@ -7,11 +7,11 @@ import { WhereClause } from './WhereClause';
  *
  * This is currently provides _very_ basic support for filtering.
  *
- * Currently, this can be either
+ * Currently, this can be either:
  *
- * 1. `{column: {operator: value}}` example: `{age: {'>': 18}}`, which translates to `age > 18`
- * 1. `{column: value}` which is equivalent to `{column: {'=': value}}`
- * 1. `{column: col('table.column')}` which translates to `column = table.column`
+ * 1. `{column: {operator: value}}` example: `{age: {'>': 18}}`, which translates to `age > 18`.
+ * 2. `{column: value}` which is equivalent to `{column: {'=': value}}`.
```

Comment:
Note that the 1. 1. 1. is intentional: Markdown is smart and will render them as 1. 2. 3. 

It's just a convenience so in case you add a new column at the second line you don't have to update 2 -> and 3 -> 4.

Doesn't really matter for small lists but for a large numbered list it starts to make a difference.

No need to change anything btw. Just an informational note.